helmfile/charts/notify-api/values.yaml

```diff
@@ -12,7 +12,7 @@ api:
   SQL_ALCHEMY_POOL_SIZE: "256"
   AWS_SES_REGION: "us-east-1"
   AWS_SES_SMTP: "email-smtp.us-east-1.amazonaws.com"
-  AWS_US_TOLL_FREE_NUMBER: "+18005555555"
+  AWS_US_TOLL_FREE_NUMBER: "+18449261787"
```

Comment:
That is the prod toll free number. The previous value seems to come from a test (this is not even the staging toll free number).